libbeat/publisher/pipeline/output_test.go

```diff
@@ -51,20 +44,27 @@ func TestMakeClientWorker(t *testing.T) {
 
 			err := quick.Check(func(i uint) bool {
 				numBatches := 300 + (i % 100) // between 300 and 399
+				numEvents := atomic.MakeUint(0)
+
+				wqu := makeWorkQueue()
+				retryer := newRetryer(logp.NewLogger("test"), nilObserver, wqu, nil)
+				defer retryer.close()
 
 				var published atomic.Uint
+				var batchCount int
```

Comment:
I used `batchCount` for debugging. We can remove it I think.

Reply:
Removed in 15a1383.